wezterm-gui/src/glyphcache.rs

```diff
@@ -745,6 +779,46 @@ impl<T: Texture2d> GlyphCache<T> {
         self.block_sprite(block)
     }
 
+    fn box_drawing_sprite(&mut self, box_drawing: BoxDrawingKey) -> anyhow::Result<Sprite<T>> {
+        let mut buffer = Image::new(
+            self.metrics.cell_size.width as usize,
+            self.metrics.cell_size.height as usize,
+        );
+        let black = SrgbaPixel::rgba(0, 0, 0, 0);
+        let white = SrgbaPixel::rgba(0xff, 0xff, 0xff, 0xff);
+
+        let cell_rect = Rect::new(Point::new(0, 0), self.metrics.cell_size);
+
+        buffer.clear_rect(cell_rect, black);
+
+        let draw_vertical = |buffer: &mut Image, x: usize| {
+            buffer.draw_line(
+                Point::new(cell_rect.origin.x + x as isize, cell_rect.origin.y),
+                Point::new(
+                    cell_rect.origin.x + x as isize,
+                    cell_rect.origin.y + self.metrics.cell_size.height,
+                ),
+                white,
+            );
+        };
+
+        use BoxDrawingKey::*;
+        match box_drawing {
+            LightVertical => draw_vertical(&mut buffer, 10),
```

Comment:
`10` here was arbitrarily chosen as a test-case. I wonder if there's (semi-)official documentation anywhere on the exact details on how to draw the different characters?

Reply:
There don't appear to be guidelines on exactly how thick the "light" variant is vs. the "heavy" variant.

The font metrics do tell us the desired height of an underline (take a look in the `line_sprite` function below: `self.metrics.underline_height`).

We also know the aspect ratio of the glyph (width-to-height ratio), so I'd suggest using the `underline_height` to compute a vertical bar thickness for the "light" variant, then eg: double that for the heavy variant.

Reply:
(and then center that line in the box)